javascript/index.js

```diff
@@ -23,18 +23,138 @@
   getInstruction("mashedPotatoes", 4, (step5) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
+  }, (error) => console.log(error)); */
 
 
 
 // Iteration 1 - using callbacks
-// ...
+
+setTimeout(() => {
```

Comment:
this is wrong... it doesn't use callbacks at all, but adjusts the timing with the number of ms waited. 

if you didn't know the amount of seconds between each step, because it would be random, how could you make the times fit? (this is a trick question - you would use callbacks to make it fit no matter the timing)

Reply:
i.e. something like this: 
```js
getInstruction("mashedPotatoes", 0, (step) => {
    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step}</li>`;
    getInstruction("mashedPotatoes", 1, (step) => {
        // and repeat...
    });
});
```

Reply:
you could even make this recursive :)